### ISSUE TITLE
Inconsistent tensor shapes in documentation for tf.nn.sampled_softmax_loss

### ISSUE THREAD

**Issue by roysmeding** [tensorflow/tensorflow#2464]
Hi,

In the documentation for tf.nn.sampled_softmax_loss (as viewed [here](https://www.tensorflow.org/versions/master/api_docs/python/nn.html#sampled_softmax_loss)) it says the equivalent full softmax probabilities can be computed using `tf.nn.softmax(tf.matmul(inputs, weights) + biases)`. However, under 'Args' it also states that `inputs` has shape `batch_size, dim` and `weights` has shape `[num_classes, dim]` — The dimensions of these tensors are incompatible for matrix multiplication.

I suspect that the _actual_ equivalent operation would be something like `tf.nn.softmax(tf.matmul(inputs, tf.transpose(weights)) + biases)` but I'm not 100% sure.

**Comment by vrv** [comment]
@girving is this just a doc bug?

**Comment by girving** [comment]
Yep, that's a doc bug.  It should be `tf.nn.softmax(tf.matmul(inputs, tf.transpose(weights)) + biases)` as suggested.  Fixing.
